Add some tips for expression dialog (#3449)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/attribute_table.rst

   |calculateField| :sup:`Field Calculator` dialog.
#. Select the |checkbox| :guilabel:`Create a new field` checkbox to save the
   calculations into a new field.

> #. Add ``length_km`` as :guilabel:`Output field name`

Reviewer: ```suggestion
#. Set :guilabel:`Output field name` to  ``length_km``
```

> #. Define the :guilabel:`Output field type` as ``Decimal number (real)``

Reviewer: ```suggestion
#. Select ``Decimal number (real)`` as :guilabel:`Output field type`
```

#. Set the :guilabel:`Output field length` to ``10`` and the :guilabel:`Precision`
   to ``3``

> #. Double click the ``$length`` function in the :guilabel:`Geometry` group

Reviewer: ```suggestion
#. Double click ``$length`` in the :guilabel:`Geometry` group to add the length of the
```

>    to add it into the Field calculator expression box.

Reviewer: ```suggestion
   geometry into the Field calculator expression box.
```

Author: not really convinced by this rewording

Reviewer: I have tried to make my proposal clearer by combining two suggestions. I have tried to avoid using "it".
